rules/crowdstrike_rules/crowdstrike_dns_request.py

```diff
@@ -1,17 +1,28 @@
+from panther_base_helpers import get_crowdstrike_field
+
 # baddomain.com is present for testing purposes. Add domains you wish to be alerted on to this list
 DENYLIST = ["baddomain.com"]
 
 
 def rule(event):
-    if event.get("DomainName") in DENYLIST:
+    # We need to run either for Crowdstrike.DnsRequest or for DnsRequest.FDREvent of 'DnsRequest'
```

Comment:
Would you mind checking the phrasing again here? Not sure if it's fully comprehensible.

Reply:
Reworded a bit. Thank you.

Reply:
Typo fixed.